composer.json

```diff
@@ -62,7 +62,10 @@
     "autoload-dev": {
         "psr-0": {
             "": "engine/tests/phpunit/"
-        }
+        },
+		"files": [
+			"engine/tests/phpunit/bootstrap.php"
```

Comment:
I don't think duplicating logic in TestCase helps us. If the suite doesn't instantiate TestCase composer won't include the class file.